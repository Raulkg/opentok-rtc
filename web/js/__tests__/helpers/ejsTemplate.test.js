var ejsTemplate = require('../../helpers/ejsTemplate.js');
test('snapshot for ejsTemplate component', () => {
  expect(ejsTemplate).toMatchSnapshot();
});