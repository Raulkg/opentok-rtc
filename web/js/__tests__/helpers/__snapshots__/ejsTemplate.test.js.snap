// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for ejsTemplate component 1`] = `
Object {
  "EJSTemplate": [Function],
}
`;
